zulipterminal/ui_tools/boxes.py

```diff
@@ -810,6 +814,40 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
         elif is_command_key("MARKDOWN_HELP", key):
             self.view.controller.show_markdown_help()
             return key
+        elif is_command_key("OPEN_EXTERNAL_EDITOR", key):
+            editor = os.environ.get("ZULIP_EDITOR_COMMAND", os.environ.get("EDITOR"))
+            if editor is None:
+                self.view.controller.report_error(
+                    "Configure $EDITOR or $ZULIP_EDITOR_COMMAND shell environment."
+                )
+                return key
+            editor_splits = editor.split(" ")
+            if len(editor_splits) >= 1:
```

Comment:
What case are you trying to avoid here? Or equally, what should happen if `len(editor_splits)<1` ?

Reply:
Agree, we only need to catch the `None` and the non existing program, the `split` will never return an empty list.